packages/features/eventtypes/lib/getPublicEvent.ts

```diff
@@ -209,6 +116,102 @@ function isAvailableInTimeSlot(
   return isWithinPeriod;
 }
 
+const getPublicEventSelect = (limitHostsToThree?: boolean) => {
+  return Prisma.validator<Prisma.EventTypeSelect>()({
+    id: true,
+    title: true,
+    description: true,
+    eventName: true,
+    slug: true,
+    isInstantEvent: true,
+    instantMeetingParameters: true,
+    aiPhoneCallConfig: true,
+    schedulingType: true,
+    length: true,
+    locations: true,
+    customInputs: true,
+    disableGuests: true,
+    metadata: true,
+    lockTimeZoneToggleOnBookingPage: true,
+    requiresConfirmation: true,
+    autoTranslateDescriptionEnabled: true,
+    fieldTranslations: {
+      select: {
+        translatedText: true,
+        targetLocale: true,
+        field: true,
+      },
+    },
+    requiresBookerEmailVerification: true,
+    recurringEvent: true,
+    price: true,
+    currency: true,
+    seatsPerTimeSlot: true,
+    seatsShowAvailabilityCount: true,
+    bookingFields: true,
+    teamId: true,
+    team: {
+      select: {
+        parentId: true,
+        metadata: true,
+        brandColor: true,
+        darkBrandColor: true,
+        slug: true,
+        name: true,
+        logoUrl: true,
+        theme: true,
+        parent: {
+          select: {
+            slug: true,
+            name: true,
+            bannerUrl: true,
+            logoUrl: true,
+          },
+        },
+        isPrivate: true,
+      },
+    },
+    successRedirectUrl: true,
+    forwardParamsSuccessRedirect: true,
+    workflows: {
+      include: {
+        workflow: {
+          include: {
+            steps: true,
+          },
+        },
+      },
+    },
+    hosts: {
+      select: {
+        user: {
+          select: userSelect,
+        },
+      },
+      ...(limitHostsToThree ? { take: 3 } : {}),
```

Comment:
We would still need all the hosts for single event type page so we don't want the limit everytime

Reply:
Wouldn't it be better to keep the variable as the number itself?

Reply:
What exactly is single event-type page?

Reply:
single event-type page is /event-types/[id] page which has assignment tab to list all members